README.md

```diff
@@ -1,7 +1,49 @@
 # market4.0-data_app_test_BE
 [![License: AGPL](https://img.shields.io/github/license/Engineering-Research-and-Development/true-connector-basic_data_app.svg)](https://opensource.org/licenses/AGPL-3.0)
 
-## Dedicated endpoint in dataApp
+## Building dataApp
+
+**Requirement:**
+
+ `Java11` `Apache Maven`
+ 
+To build dataApp you will have to do one of the following:
```

Comment:
Does this only reffer to the bulletin or bulletin and GH Packages solution?

Reply:
Added Solution 1 and Solution 2, to make difference between those 2